config/config-features.yaml

```diff
@@ -0,0 +1 @@
+core/configmaps/features.yaml
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
core/configmaps/features.yaml

```

Reply:
This is a symlink. It is not an issue.